app/test/unit/app/Docs.test.jsx

```diff
@@ -0,0 +1,325 @@
+import React from 'react'
```

Comment:
This test goes through all the docs routes with and without the flag and checks that something is shown.